File: app/src/main/java/ru/playsoftware/j2meloader/base/BaseActivity.java
package ru.playsoftware.j2meloader.base;

import android.annotation.SuppressLint;
import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import ru.playsoftware.j2meloader.R;

@SuppressLint("Registered")
public class BaseActivity extends AppCompatActivity {

	@Override
	protected void onCreate(@Nullable Bundle savedInstanceState) {
		if (getSupportActionBar() != null) {
			getSupportActionBar().setElevation(getResources().getDisplayMetrics().density * 2);
		}
		super.onCreate(savedInstanceState);
	}
}

File: app/src/main/java/app/activities/BaseActivity.kt
package app.activities

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

@SuppressLint("Registered")
open class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        if (supportActionBar != null) {
            supportActionBar!!.elevation = resources.displayMetrics.density * 2
        }
        super.onCreate(savedInstanceState)
    }
}